kratos/python_scripts/modeler_factory.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics as KM
+
+# Other imports
+from importlib import import_module
+
+class KratosModelerFactory(object):
+    def ConstructListOfModelers( self, model, modeler_list ):
+        constructed_modelers = []
+        for modeler_item in modeler_list:
+            if modeler_item.Has("modeler_name"):
+                modeler_name = modeler_item["modeler_name"].GetString()
+                if (KM.HasModeler(modeler_name)):
+                    constructed_modelers.append( KM.CreateModeler(modeler_name, modeler_item["Parameters"]) )
+
+                else:
+                    if modeler_item.Has("kratos_module"):
+                        """Location of the modelers in Kratos; eg.:
+                        - KratosMultiphysics
+                        - KratosMultiphysics.FluidDynamicsApplication
+                        """
+                        kratos_module_name = modeler_item["kratos_module"].GetString()
+
+                        if not kratos_module_name.startswith("KratosMultiphysics"):
+                            kratos_module_name = "KratosMultiphysics." + kratos_module_name
+                        modeler_name = kratos_module_name + "." + modeler_name

```

Comment:
```suggestion
                        modeler_name = kratos_module_name + ".modelers." + modeler_name

```

Reply:
this way the modellers would be placed in `kratos/python_scripts/modelers/` thus bringing more structure to this folder

Reply:
I agree, that would mean that also applications need to follow this.